### DISCUSSION TITLE
Ubuntu boots sometimes and then freezes on the boot page

### DISCUSSION THREAD

**Discussion by sai-srinivasan-v** [utmapp/UTM#6583] [Linux]
**Describe the issue**  
I have Ubuntu 20.04 booted from https://mac.getutm.app/gallery/ubuntu-20-04 it can sometimes start up and work the way i intend it to and sometimes it freezes on the start page indefinitely. I have a lot of my projects content stored in that Machine and if it doesnt boot up i am done for.  

**Configuration**  
* UTM Version:  Version 4.5.3 (99)
* macOS Version: macOS Sonoma Version 14.0 (23A344)
* Mac Chip (Intel, M1, ...): M2 Pro

**Debug log**  
Debug Log has been attached below

**Upload VM**  

[Uploading config.plist.zip…]()

[debug.log](https://github.com/user-attachments/files/15937583/debug.log)

**Comment by sai-srinivasan-v** [comment]
<img width="799" alt="Screenshot 2024-06-22 at 10 55 20 PM" src="https://github.com/utmapp/UTM/assets/55703998/c149e966-2051-4280-89f1-ebf3ecdb309b">
This is the boot window that remains indefinitely

**Comment by osy** [comment]
Your Ubuntu installation may have been corrupted. Try booting from a rescue ISO or to recovery mode.
